account_fiscal_position_partner_type/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- encoding: utf-8 -*-
+# Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import account_fiscal_position
+from . import res_partner
+from . import res_company
```

Comment:
Putting this order works perfectly:

```suggestion
from . import res_company  # This one should be first
from . import account_fiscal_position
from . import res_partner
```

Reply:
@pedrobaeza The problem is that Travis doesn't pass with this configuration. I need to create a hook so that Travis doesn't generate an error, although I haven't managed to understand the final reason for the error.

Reply:
Let me check. It's a collision with other module, as installed alongside it's working.